components/Rating/rating.css

```diff
@@ -0,0 +1,40 @@
+.ratings-wrapper {
+	align-items: center;
+	align-self: center;
+	background-color: var(--white);
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	justify-self: center;
```

Comment:
I thing this property is not in use here as u are giving it in wrapper itself, and justify content already making it center! please correct me if i m wrong.

Reply:
Ohh yes! that's right. My bad. I'll remove it. Thanks for the review.